rfcs/0000_conducting_research_while_building_a_stable_platform.md

```diff
@@ -0,0 +1,143 @@
+| _Failure_ | high / tolerated / assumed | moderate / unsurprising but disappointed | not tolerated, focus on constant improvement / operational efficiency and surprised by failure |
+| _Market action_ | gambling / driven by gut | gambling / driven by gut | exploring "found" value / market analysis / listening to customers |
+| _Efficiency_ | reducing the cost of change (experimentation and prototyping) | reducing the cost of waste (learning and stability) | reducing the cost of waste (learning and stability) / reducing the cost of deviation (volume operations) |
+| _Decision drivers_ | culture / analysis & synthesis | culture / analysis & synthesis | analysis & synthesis / previous experience |
+
+While the characteristics of the platform in the future will be quite different from ongoing research in general (which we're planning for), today's platform already differs in two significant aspects: _Failure_ and _Efficiency_.
+
+### Failure
+
+Because of the nature of research, we expect experiments to not work, at least initially. Our tolerance for failure is high. For a platform to be useful, it cannot fail at the same rate as research. While initially, users may be willing to tolerate some failure due to our growing pains, ultimately, the goal is for failure to become surprising and not generally tolerated.
+
+### Efficiency
+
+How we organize our activity can have consequences at how successful we are in execution.
+
+For research, we want to be able to minimize the cost of change. In research, ideas can change rapidly. For an example, one can trace a history and the timeline of research into [Modeling Object Behaviors](https://www.youtube.com/playlist?list=PLXpTU6oIscrn_v8pVxwJKnfKPpKSMEUvU). We want to be able to rapidly try new experiments, new theories, new implementations. We want minimal friction between an idea and experimental data. Since experiments can have a high failure rate, when we write prototype code, it is possible that most or some of it will be thrown away or significantly altered based on what we learn from the experiments.
+
+For a platform, there is only so much change that platform users can absorb. While we intend for the platform to always stay up to date with the latest capabilities discovered through research, those capabilities need a level of stability before being incorporated into the platform. Even if we were to rapidly iterate on platform internals, any externally facing interface benefits from being stable. While significant improvements make changes easier to adopt, in general, the platform users are focused on doing their own research and/or solving their own business problems and want to minimize the cost of using our platform in toil, time, and treasure.
+
+### Conflicting Goals
+
+So, our challenge is to navigate the conflicting goals of research and platform development. Research is the source of new capabilities. Platform is the stability on which solutions are built.
```

Comment:
```suggestion
So, our challenge is to navigate the conflicting goals of research and platform development. Research is the source of new capabilities. Platform is the ease-of-use and stability on which solutions are built.
```
Not sure how to best rephrase this, but just wanted to highlight that stability isn't the only purpose of the platform. It should also provide an easier to use and understand interface for a user that does not understand all of Monty's internal mechanics.

Reply:
I tried `Platform, on which solutions are built, is usable and stable.`